exercises/practice/bowling/src/test/kotlin/BowlingGameTest.kt

```diff
@@ -0,0 +1,214 @@
+import org.junit.jupiter.api.assertThrows
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class BowlingGameTest {
+    private fun playGame(rolls: IntArray): BowlingGame {
+        val game = BowlingGame()
+        rolls.forEach {
+            game.roll(it)
+        }
+        return game
+    }
+
+    @Test
+    fun `should be able to score a game with all zeros`() {
+        val game = playGame(intArrayOf(0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
+        assertEquals(0, game.score())
+    }
+
+    @Test
```

Comment:
All but the first test should be ignored by default.

```suggestion
    @Test
    @Ignore
```

Reply:
Corrected